Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -301,17 +301,25 @@ def __init__(self, url, dev_branches=None, prod_branches=None, contributors=None
         self.pull_requests = self.PRGenerator(self)
 
     def credentials(self, required=True):
-        username, token = credentials(
+        # FIXME: Should use webkitbugspy's impelementation
+        args = dict(
             url=self.api_url,
             required=required,
             name=self.url.split('/')[2].replace('.', '_').upper(),
             prompt='''GitHub's API
-Please generate a 'Personal access token' via 'Developer settings' with 'repo' and 'workflow' access
-for your {} user'''.format(self.url.split('/')[2]),
+Please generate a 'Personal access token' via 'Developer settings' with 'repo' and 'workflow'
```

Comment:
same feedback here.
Also, why is this code duplicated? (I do see a FIXME, so that's fine to fix in later PR, just trying to understand the reason if we need it at both places currently)

Reply:
At the moment, `webkitscmpy` (library that handles source code management) does not depend on `webkitbugspy` (library that handles interaction with bug trackers), although it will, and `webkitbugspy` will never depend on `webkitscmpy`. Basically, `webkitbugspy` is a super new library (as in it was committed for the first time in the last week) and before `webkitscmpy` actually uses it, I would `webkitbugspy` to have the things that `webkitscmpy` needs.